src/setStateIndicator/randomNum.js

```diff
@@ -12,4 +12,5 @@ export const setStateDependingRandomNum = (maxNum, setIndicator) => {
       return;
     }
   }, time_between_run_function_getRandomNumber_ms);
+  return timerId;
```

Comment:
timerId could be a `const` because it does not change. `const` should be your default choice for a variable